guides/developers-guide/concepts/dev-how-do-i.md

```diff
@@ -0,0 +1,255 @@
+# For Developers: How Do I...?
```

Comment:
Almost all of this stuff is duplicates from what I already added in the How-to guide. Can you de-duplicate and resubmit?

Reply:
Done